inspire_dojson/hep/rules/bd7xx.py

```diff
@@ -72,6 +72,12 @@ def _get_material(value):
         if normalized_m_value in valid_materials:
             return normalized_m_value
 
+    def _get_parent_isbn(value):
+        z_value = force_single_element(value.get('z', ''))
+        normalized_z_value = z_value.replace('-', '').upper()
```

Comment:
this is not sufficient: we need to normalize all ISBNs to ISBN13 (if they are valid). Problem is that it will change citation counts when sending back to legacy the normalized ISBN. So maybe we should wait for the reference matcher on labs?

Reply:
Yeah, now I remember our conversation from yesterday... BTW, this was too simple, I amended it to use `ISBN` from `isbnid_fork`. Let me check if it's already doing what it should on ISBN-10.

Reply:
It does.